consensus/src/liveness/leader_reputation.rs

```diff
@@ -29,6 +29,9 @@ pub struct LibraDBBackend {
     window: Mutex<Vec<(u64, NewBlockEvent)>>,
 }
 
+// REVIEW: I couldn't understand how this worked from the comments. For example, how do you make sure that
+// for round R, everybody is using the same view (even if they are synced up to different points)
```

Comment:
I should have filled more comments for this, in short we can't guarantee everybody is using the same view(committed version) for round R.
In such cases, they'll time out and do a n^2 sync and hopefully next round they'd have the same view.